doc/making_releases.md

```diff
@@ -12,6 +12,14 @@ NOTE: it is untested on Windows
 After the script runs it places the created releases in the `builds`
 folder.
 
+## Installing export templates
+Godot's export templates must be installed to export the project. Instructions
+can be found here: 
+https://docs.godotengine.org/en/stable/tutorials/export/exporting_projects.html?highlight=export%20templates
+
+Alternatively, you can run the script GodotTemplateInstaller.cs in the scripts
+folder of this repository.
```

Comment:
This should mention the exact command: `dotnet run --project Scripts -- godot-templates` which is how you run that script.